mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
Not yet, some work in this direction has started. It's already possible for the note input toolbar. See Edit > Edit toolbars. See also https://musescore.org/en/node/113736